doc/api/tty.md

```diff
@@ -88,6 +88,10 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* Terminal window resize events are unreliable on Windows. The event 
+handler is only executed for changes in the width of the window and the 
```

Comment:
@bzoz that seems like a lot of information, could you suggest the exact wording you want?